pkg/plugins/golang/v3/init.go

```diff
@@ -171,12 +172,23 @@ func checkDir() error {
 			if strings.HasPrefix(info.Name(), ".") {
 				return nil
 			}
+			// Allow files ending with '.md' extension
+			if strings.HasSuffix(info.Name(), ".md") && !info.IsDir() {
+				return nil
+			}
+			// Allow capitalized files (like LICENSE, CODEOWNERS..)
+			for index, l := range info.Name() {
+				if !unicode.IsUpper(l) || info.Name() == "PROJECT" {
+					break
+				}
+				if index == len(info.Name())-1 {
+					return nil
+				}
```

Comment:
I feel like it is more natural to have a flag set before the loop that gets changed if any non-uppercase is found and returning outside of the loop if the flag has changed than to check this condition every loop.

Reply:
Fixed!